__tests__/app.test.js

```diff
@@ -112,3 +113,65 @@ describe("/api/articles/:article_id", () => {
     });
   });
 });
+
+describe("/api/articles/:article_id/comments", () => {
+  describe("GET", () => {
+    test("200: Should respond with comment objects with relevent properties for a passed articel parameter", () => {
+      const id = 3;
+
+      return request(app)
+        .get(`/api/articles/${id}/comments`)
+        .expect(200)
+        .then(({ body }) => {
+          const commentsArr = body.comments;
+
+          expect(Array.isArray(commentsArr)).toBe(true);
+          expect(commentsArr.length).toBe(2);
+
+          commentsArr.forEach((comment) => {
+            expect(comment).toHaveProperty("comment_id");
+            expect(comment).toHaveProperty("votes");
+            expect(comment).toHaveProperty("created_at");
+            expect(comment).toHaveProperty("author");
+            expect(comment).toHaveProperty("body");
+            expect(comment).toHaveProperty("article_id", id);
+          });
+        });
+    });
+
+    test("200: Should respond with empty for a valid article with no comments", () => {
+      const id = 8;
+
+      return request(app)
+        .get(`/api/articles/${id}/comments`)
+        .expect(200)
+        .then(({ body }) => {
+          const commentsArr = body.comments;
+
+          expect(commentsArr).toEqual([]);
+        });
+    });
+
+    test("200: Should respond with most recent comments first", () => {
+      const id = 3;
+
+      return request(app)
+        .get(`/api/articles/${id}/comments`)
+        .expect(200)
+        .then(({ body }) => {
+          const commentsArr = body.comments;
+
+          expect(commentsArr).toBeSortedBy("created_at", { descending: true });
+        });
+    });
+
+    test("404: Should respond with not found for an article id not in the db", () => {
+      const id = 999;
+
+      return request(app)
+        .get(`/api/articles/${id}/comments`)
+        .expect(404)
+        .then(({ body }) => {});
```

Comment:
Should this not check the error message coming back?

Reply:
Yeah totally missed that cheers